docs/general/recording-errors.md

```diff
@@ -5,17 +5,25 @@
 <!-- toc -->
 
 - [What constitutes an error](#what-constitutes-an-error)
+- [Error vs exception](#error-vs-exception)
 - [Recording errors on spans](#recording-errors-on-spans)
 - [Recording errors on metrics](#recording-errors-on-metrics)
+- [Recording errors on logs and events](#recording-errors-on-logs-and-events)
 - [Recording exceptions](#recording-exceptions)
 
 <!-- tocstop -->
 
 This document provides recommendations to semantic convention and instrumentation authors
-on how to record errors on spans and metrics.
+on how to record errors on spans, metrics, and logs.
 
 Individual semantic conventions are encouraged to provide additional guidance.
 
+In the scope of this document, the terms error and exception are defined as follows:
+
+- *Error* refers to a general concept describing any non-success condition. This may include an, non-success status code, or an invalid response.
```

Comment:
```suggestion
- *Error* refers to a general concept describing any non-success condition. This may include a non-success status code, or an invalid response.
```